Polaris/src/main.cpp

```diff
@@ -1,14 +1,18 @@
+#include <Controls/EKF/EKF.h>
```

Comment:
Riley didn't kill the other EKF, but it is still here, technically redundant. Also files could just be killed. Also also Dan said EKF wasn't working.